src/main/java/uk/gov/cslearning/catalogue/service/record/LearnerRecordService.java

```diff
@@ -20,7 +19,7 @@
 @Service
 public class LearnerRecordService {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(IdentityTokenServices.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(LearnerRecordService.class);
```

Comment:
Weird, to reference the IdentityTokenServices class here that must have been a copy and paste error from whenever this class was created.